src/main/java/longah/Command.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * The Command class parses user input into a keyword and arguments.
+ */
+public class Command {
```

Comment:
Might be neater to implement this as `abstract` and extract commands to separate classes.